app/model/editions/EditionsCollection.scala

```diff
@@ -0,0 +1,59 @@
+package model.editions
+
+import java.time.ZonedDateTime
+import play.api.libs.json.Json
+
+import scalikejdbc.WrappedResultSet
+
+case class EditionsCollection(
+    id: String,
+    name: String,
```

Comment:
Any reason this isn't `displayName` like on an existing collection? - Sorry for the spam, I'm just comparing the two models!